src/lib/utils/LoginUtils.tsx

```diff
@@ -0,0 +1,48 @@
+import { SynapseClient } from '.'
```

Comment:
Two small things: 
1. this file can be made a .ts file instead of .tsx since there's no jsx
2. I can't recall if I had an issue with this in the past but the build process might not like this _import from '.'_ syntax, instead it might want _import from './index'_